blueprints/codes/eurocode/nen_en_1993_5_2008/chapter_5_ultimate_limit_states/formula_5_6.py

```diff
@@ -0,0 +1,56 @@
+"""Formula 5.6 from NEN-EN 1993-5:2008 Chapter 5 - Ultimate limit state."""
+
+from blueprints.codes.eurocode.nen_en_1993_5_2008.chapter_5_ultimate_limit_states import NEN_EN_1993_5_2008
+from blueprints.codes.formula import Formula
+from blueprints.codes.latex_formula import LatexFormula
+from blueprints.type_alias import MM, MM2
+from blueprints.validations import raise_if_less_or_equal_to_zero
+
+
+class Form5Dot6ProjectedShearArea(Formula):
+    """Class representing formula 5.6 for the projected shear area."""
+
+    label = "5.6"
+    source_document = NEN_EN_1993_5_2008
+
+    def __init__(
+        self,
+        h: MM,  # Overall height
+        tf: MM,  # Flange thickness
+        tw: MM,  # Web thickness
```

Comment:
```suggestion
        t_f: MM,  # Flange thickness
        t_w: MM,  # Web thickness
```

Try to keep the style consistent